packages/block-library/src/search/style.scss

```diff
@@ -1,17 +1,54 @@
 .wp-block-search {
-	display: flex;
-	flex-wrap: wrap;
+	.wp-block-search__inside-wrapper {
+		display: flex;
+		flex: auto;
+		flex-wrap: nowrap;
+		max-width: 100%;
+	}
 
 	.wp-block-search__label {
 		width: 100%;
 	}
 
 	.wp-block-search__input {
 		flex-grow: 1;
-		max-width: 360px;
+		min-width: 50px;
+		border: 1px solid $dark-gray-200;
 	}
 
 	.wp-block-search__button {
 		margin-left: 10px;
+		word-break: normal;
+
+		svg {
+			min-width: 25px;
+			min-height: 25px;
```

Comment:
Recently all front-facing styles were converted from using pixel values to `em`.
Not a blocker, we can always add an additional commit to convert these too, but it would be nice if we got it right from the start. 
All `px` values here - except borders - should be converted to `em`